pkg/webhook/request.go

```diff
@@ -28,21 +29,21 @@ import (
 	"github.com/banzaicloud/koperator/pkg/util"
 
 	"github.com/banzaicloud/koperator/api/v1alpha1"
+	"github.com/banzaicloud/koperator/api/v1beta1"
 )
 
 var (
-	kafkaTopic = reflect.TypeOf(v1alpha1.KafkaTopic{}).Name()
+	kafkaTopic   = reflect.TypeOf(v1alpha1.KafkaTopic{}).Name()
+	kafkaCluster = reflect.TypeOf(v1beta1.KafkaCluster{}).Name()
 )
 
 func (s *webhookServer) validate(ar *admissionv1.AdmissionReview) *admissionv1.AdmissionResponse {
 	req := ar.Request
-
 	l := log.WithValues("kind", req.Kind, "namespace", req.Namespace, "name", req.Name, "uid", req.UID,
 		"operation", req.Operation, "user info", req.UserInfo)
-	l.Info("AdmissionReview")
-
 	switch req.Kind.Kind {
 	case kafkaTopic:
+		l.Info("AdmissionReview")
```

Comment:
Instead of `l.Info("AdmissionReview")` you could initialize the logger as  
```
l := log.WithName("AdmissionReview").WithValues("kind", req.Kind, "namespace", req.Namespace, "name", req.Name, "uid", req.UID,
 		"operation", req.Operation, "user info", req.UserInfo)
```